frame/contracts/fixtures/account_entrance_count_call.wat

```diff
@@ -0,0 +1,37 @@
+;; This fixture tests if seal_account_entrance_count works as expected
```

Comment:
```suggestion
;; This fixture tests if account_entrance_count works as expected
```
We removed `seal_` prefix for new host functions. Please, correct this in all places for the host functions you've added.